moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -71,7 +67,7 @@ const ProjectTeamTable = ({
    */
   const tuplesContain = (tupleList, tupleItem) =>
     !!tupleList.find(
-      currentTuple =>
+      (currentTuple) =>
```

Comment:
I think the lint rules changed, because now it wants `()` around single variables

Reply:
so many parentheses!

Reply:
glad it wasn't just me—this was throwing me off when i updated prettier in the react-scripts upgrade